test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -29,30 +29,34 @@ public class IrisExample
         public void FirstExperienceWithML()
         {
             // Load the data into the system.
-            string dataPath = "iris-data.txt";
-            var data = TextReader.FitAndRead(env, dataPath, row => (
-                Label: row.ReadString(0),
-                SepalWidth: row.ReadFloat(1),
-                SepalLength: row.ReadFloat(2),
-                PetalWidth: row.ReadFloat(3),
-                PetalLength: row.ReadFloat(4)));
-
-
-            var preprocess = data.Schema.MakeEstimator(row => (
-                // Convert string label to key.
-                Label: row.Label.DictionarizeLabel(),
-                // Concatenate all features into a vector.
-                Features: row.SepalWidth.ConcatWith(row.SepalLength, row.PetalWidth, row.PetalLength)));
-
-            var pipeline = preprocess
-                // Append the trainer to the training pipeline.
-                .AppendEstimator(row => row.Label.PredictWithSdca(row.Features))
-                .AppendEstimator(row => row.PredictedLabel.KeyToValue());
-
-            // Train the model and make some predictions.
-            var model = pipeline.Fit<IrisExample, IrisPrediction>(data);
-
-            IrisPrediction prediction = model.Predict(new IrisExample
+            string dataPath = "iris.txt";
+            var dataReader = TextReader.CreateReader(env, dataPath, row => (
+                label: row.LoadText(0),
+                sepalWidth: row.LoadFloat(1),
+                sepalLength: row.LoadFloat(2),
+                petalWidth: row.LoadFloat(3),
+                petalLength: row.LoadFloat(4)));
+
+
+            var pipeline = dataReader.MakeEstimator()
+                .Append(row => (
+                    // Convert string label to key.
+                    label: row.Label.DictionarizeLabel(),
+                    // Concatenate all features into a vector.
+                    features: row.SepalWidth.ConcatWith(row.sepalLength, row.petalWidth, row.petalLength)))
+                .Append(row => row.label.PredictSdcaMultiClass(row.features))
```

Comment:
>label.PredictSdcaMultiClass [](start = 35, length = 27)

@Zruty0, @tfinley@gmail.com i feel like calling PredictXXX on the  label, is a bit counter-intuitive. 

Predict is supposed to return the Label. 
It makes more sense to me to call PredictXXX on the model, and have the label, as currently is, returned as one of the tuple elements.